code/modules/research/designs.dm

```diff
@@ -604,7 +604,45 @@
 	build_path = /obj/item/weapon/gun/energy/cutter/plasma
 	sort_string = "TAEAC"
 
-
+/datum/design/item/weapon/divet
+	name = "Winchester NK Divet pistol"
+	id = "divet"
+	req_tech = list(TECH_COMBAT = 4, TECH_MATERIAL = 5)
+	materials = list(MATERIAL_STEEL = 10000, MATERIAL_SILVER = 4000)
+	build_path = /obj/item/weapon/gun/projectile/divet
+	sort_string = "TACEA"
+
+/datum/design/item/weapon/divetslug
+	name = "divet magazine"
+	id = "divetslug"
+	req_tech = list(TECH_COMBAT = 4, TECH_MATERIAL = 4)
```

Comment:
Requirements should be 1 and 2.

Reply:
The department starts off with a combat tech of 1, that seems off to me as it would mean by deconstructing a matter bin you could create divet slugs roundstart, which are rather costly at cargo, the intention here was that in order to get them you'd have to boost your combat tech considerably, which usually would use many materials vital to constructing them, such as steel, bringing up the net cost.

Reply:
Alright, my apologies. Make 2 and 3.